src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -501,4 +565,267 @@ describe('atomicallyReadWriteResources', () => {
             });
         }
     });
+
+    describe('Bundle transaction with multiple resources', () => {
+        async function runTransaction(
+            useVersionedReferences: boolean,
+            supportUpdateCreate: boolean,
+            errorFindingLatestVersion: boolean,
+        ) {
+            // BUILD
+            const transactWriteItemSpy = sinon.spy();
+            AWSMock.mock('DynamoDB', 'transactWriteItems', (params: TransactWriteItemsInput, callback: Function) => {
+                transactWriteItemSpy(params);
+                callback(null, {});
+            });
+            // new organization, create
+            const managingOrgId = 'org1';
+            const managingOrgResource = {
+                id: managingOrgId,
+                resourceType: 'Organization',
+                name: 'New Org .Inc',
+                meta: { lastUpdated: new Date().toISOString() },
+            };
+            // this org already exists, just referencing to it
+            const contactOrgId = 'org2';
+            const contactOrgResource = {
+                id: contactOrgId,
+                resourceType: 'Organization',
+                name: 'Existing Ltd.',
+                meta: { versionId: '7', lastUpdated: new Date().toISOString() },
+            };
+            // practitioner already exists but we also update it now
+            const practitionerId = 'practitioner1';
+            const oldPractitionerResource = {
+                id: practitionerId,
+                resourceType: 'Practitioner',
+                name: 'James Brown',
+                meta: { versionId: '2', lastUpdated: new Date().toISOString() },
+            };
+            // updated version included in the bundle
+            const newPractitionerResource = {
+                id: practitionerId,
+                resourceType: 'Practitioner',
+                name: 'James Dean',
+                meta: { versionId: '3', lastUpdated: new Date().toISOString() },
+            };
+            // new patient resource with 3 references
+            const patientResource = {
+                id,
+                resourceType: 'Patient',
+                name: [
+                    {
+                        family: 'Jameson',
+                        given: ['Matt'],
+                    },
+                ],
+                managingOrganization: {
+                    reference: 'Organization/org1',

```

Comment:
Could we have 1 test case of an absolute URL? ie:
```suggestion
                    reference: '${apiUrl}/Organization/org1',

```